### DISCUSSION TITLE
After recent update - random meshes showing up below model

### DISCUSSION THREAD

**Discussion by teammixtive** [google/model-viewer#2395] [Q&A]
Hi, 

Just a day ago, some models that I displayed in model-viewer were working perfectly. Suddenly. without doing anything to the models, some random squares are appearing below the model as in the image below. Any ideas why? 

<img width="678" alt="Screen Shot 2021-05-18 at 11 48 27" src="https://user-images.githubusercontent.com/72607050/118631734-134fa780-b7d0-11eb-9e9a-2a65a0d0ad23.png">

**Comment by elalish** [comment]
Pretty certain this is the same issue as #2386, which has a work-around and also just got fixed. We may put out a point release for this, but need to check on some other bugs first.
